README.md

```diff
@@ -167,8 +161,6 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
 I would like to thank...
```

Comment:
- [ ] [ OPTIONAL ]  This PR should be invalidated by the previous reviewer because the Acknowledgments section is empty with placeholders you should never leave your readme file empty with placeholders so that it will be pro